qcodes/instrument_drivers/QuTech/D5a.py

```diff
@@ -0,0 +1,120 @@
+from qcodes import Instrument
+from qcodes.utils.validators import Enum, Numbers
+
+try:
+    from spirack import D5a_module
+except ImportError:
+    raise ImportError(('The D5a_module class could not be found. '
+                       'Try installing it using pip install spirack'))
+
+from functools import partial
+
+
+class D5a(Instrument):
+    """
+    Qcodes driver for the D5a DAC SPI-rack module.
+
+    functions:
+    -   set_dacs_zero   set all DACs to zero voltage
+
+    parameters:
+    -   dacN:       get and set DAC voltage
+    -   stepsizeN   get the minimum step size corresponding to the span
+    -   spanN       get and set the DAC span: '4v uni', '4v bi', or '2.5v bi'
+
+    where N is the DAC number from 1 up to 16
+
+    """
+
+    def __init__(self, name, spi_rack, module, inter_delay=0.1, dac_step=10e-3,
+                 reset_voltages=False, mV=False, **kwargs):
+        """ Create instrument for the D5a module.
+
+        The D5a module works with volts as units. For backward compatibility
+        there is the option to allow mV for the dacX parameters.
+
+        Args:
+            name (str): name of the instrument.
+
+            spi_rack (SPI_rack): instance of the SPI_rack class as defined in
+                the spirack package. This class manages communication with the
+                individual modules.
+
+            module (int): module number as set on the hardware.    
+            inter_delay (float): time in seconds, passed to dac parameters of the object
+            dac_step (float): max step size (V or mV), passed to dac parameters of the object
+            reset_voltages (bool): passed to D5a_module constructor    
```

Comment:
In this line and the three lines above it, there is trailing whitespace and too long lines.